packages/wpcom.js/docs/site.md

```diff
@@ -303,3 +303,33 @@ site.statsVisits( function ( err, data ) {
 	// data visits response
 } );
 ```
+
+## Email Stats
+
+### Site#emailStatsAlltime([query, ]fn)
```

Comment:
It seems like the function signature is missing `postId` & `statType` (and includes query, which is not there anymore).